provider/cluster/kube/builder.go

```diff
@@ -293,6 +296,34 @@ func (b *deploymentBuilder) container() corev1.Container {
 	return kcontainer
 }
 
+const envVarAkashGroupSequence = "AKASH_GROUP_SEQUENCE"
+const envVarAkashDeploymentSequence = "AKASH_DEPLOYMENT_SEQUENCE"
+const envVarAkashOrderSequence = "AKASH_ORDER_SEQUENCE"
+const envVarAkashOwner = "AKASH_OWDER"
+const envVarAkashProvider = "AKASH_PROVIDER"
+const envVarAkashClusterPublicHostname = "AKASH_CLUSTER_PUBLIC_HOSTNAME"
```

Comment:
use
```go
const (
  foo = "val"
  bar = "val"
)
```
format please

Reply:
I need to get our code linter to prefer this, I keep forgetting it.